lib/plugin.client.js

```diff
@@ -10,7 +10,13 @@ import { Integrations as TracingIntegrations } from '@sentry/tracing'
 export default function (ctx, inject) {
   <% if (options.initialize) { %>
   /* eslint-disable object-curly-spacing, quote-props, quotes, key-spacing, comma-spacing */
-  const config = <%= serialize(options.config) %>
+  const config = <%= JSON.stringify(options.config) %>
```

Comment:
What's the reason for this change?

Reply:
No reason at all, it's a mistake due to the rebase. I'll fix it.